### PR TITLE
TLC - 286

### PR DESCRIPTION
**Pull request by Edemina**
files changed: 
login.page - added and changed several selectors
loginLayOut.spec 
testResult/login 
https://jira.pasv.us/browse/TLC-286 - Create autotests for Login Page / layout

### CODE REVIEW

File: pageObjects/login.page.js [hunk 1]
```diff
@@ -12,20 +12,28 @@ class LoginPage extends BasePage {
...
   }
 
   get loginGoogleBtn() {
-    return $$('#firebaseui_containerbutton')[0];
+    return $$('.firebaseui-list-item')[0];
   }
 
   get loginFacebookBtn() {
-    return $$('#firebaseui_containerbutton')[1];
+    return $$('.firebaseui-list-item')[1];
```
Comment (aliakseyevavikt) [inline pageObjects/login.page.js:36]: Is it better to use $('.firebaseui-idp-facebook')?

File: pageObjects/login.page.js [hunk 2]
```diff
@@ -12,20 +12,28 @@ class LoginPage extends BasePage {
...
+    return $('.anticon-lock');
+  }
+
   get loginBtn() {
     return $('button[type="submit"]');
   }
 
   get loginGoogleBtn() {
-    return $$('#firebaseui_containerbutton')[0];
+    return $$('.firebaseui-list-item')[0];
```
Comment (aliakseyevavikt) [inline pageObjects/login.page.js:32]: Is it better $('.firebaseui-idp-google ')?

File: pageObjects/login.page.js (hunk 1 above)
Comment (Edemina) [reply pageObjects/login.page.js]: thanks for the advice. I changed and pushed the updates

File: pageObjects/login.page.js (hunk 2 above)
Comment (Edemina) [reply pageObjects/login.page.js]: thanks for the advice. I changed and pushed the updates

**Pull request merged**